server/src/main/scala/org/http4s/server/middleware/ChunkAggregator.scala

```diff
@@ -8,19 +8,20 @@ import cats.effect.Sync
 import cats.implicits._
 import fs2._
 import org.http4s.headers._
+import org.http4s.headers._
 
 object ChunkAggregator {
   def apply[F[_]: FlatMap, G[_]: Sync, A](f: G ~> F)(
       @deprecatedName('service) http: Kleisli[F, A, Response[G]]): Kleisli[F, A, Response[G]] =
     http.flatMapF { response =>
       f(
         response.body.chunks.compile
-          .fold((Segment.empty[Byte], 0L)) {
-            case ((seg, len), c) => (seg ++ c.toSegment, len + c.size)
-          }
-          .map {
-            case (body, len) =>
-              removeChunkedTransferEncoding[G](response.withBodyStream(Stream.segment(body)), len)
+          .to[Vector]
```

Comment:
`.to[Array].map(vec=> Chunk.bytes(vec))`?
I'm hoping `CanBuildFrom` uses an arraybuffer or something.